dronecan/sensors/rc/20400.RCInput.uavcan

```diff
@@ -0,0 +1,6 @@
+uint8 STATUS_RSSI_VALID = 1 # RSSI field is valid
+uint8 STATUS_FAILSAFE = 2 # receiver has lost contact with transmitter
+uint16 status
+
+uint8 rssi
+uint12[<=24] rcin
```

Comment:
Thinking about the future we already support 32 servos in AP. And  just because right now most receivers only receive a limited number of channels doesn't mean they always will...

```suggestion
uint12[<=24] rcin
uint8 index # Index of the first rc_channel in the array, index 0 is rc_channel at array indices 0 - 23, index 24 is rc_channel at array indices 24 - 47, etc.
```

Reply:
> Thinking about the future we already support 32 servos in AP. And just because right now most receivers only receive a limited number of channels doesn't mean they always will...

Overkill :-)  Most people aren't using 16 channels, let alone 24 :-)